pkg/ctl/cmdutils/configfile.go

```diff
@@ -481,9 +483,16 @@ func normalizeNodeGroup(ng *api.NodeGroup, l *commonClusterConfigLoader) error {
 		return fmt.Errorf("%s volume type is not supported via flag --node-volume-type, please use a config file", api.NodeVolumeTypeIO1)
 	}
 
+	normalizeBaseNodeGroup(ng, l.CobraCommand)
 	return nil
 }
 
+func normalizeBaseNodeGroup(np api.NodePool, cmd *cobra.Command) {
```

Comment:
not really sure what `normalize` means 🤔  possible to have a more descriptive func name?

Reply:
I couldn't think of anything better, and we already had a `normalizeNodeGroup` function here. I'm open to suggestions :slightly_smiling_face:

Reply:
I am always confused by the `normalizeNodeGroup` func! :sob:

😫  neither can I